src/components/SelectionList/BaseListItem.tsx

```diff
@@ -66,8 +75,10 @@ function BaseListItem<TItem extends ListItem>({
                     <>
                         <View style={wrapperStyle}>
                             {canSelectMultiple && (
-                                <View
+                                <PressableWithFeedback
+                                    accessibilityLabel={item.text}
```

Comment:
yes, I noticed that as well. Btw not blocker since admin cannot be actually removed.
Hope @ArekChr would fix it as follow-up along with my NAB feedback